docs/public-networks/reference/api/index.md

```diff
@@ -976,6 +976,88 @@ curl -X POST --data '{"jsonrpc":"2.0","method":"debug_getRawHeader","params":["0
 
 <!--/tabs-->
 
+### `debug_getRawReceipts`
+
+Returns the [RLP encoding](https://ethereum.org/en/developers/docs/data-structures-and-encoding/rlp/)
+of the transaction receipts of the specified block.
+
+#### Parameters
+
+`blockNumber`: _string_ - integer representing a block number or one of the string tags `latest`,
```

Comment:
This applies throughout for the `blockNumber` parameter. Should we say, "hexadecimal integer representing..."?
Also, should we mention `finalized` and `safe` as possible options?

Could be implemented in separate PR if you agree. Would be a "good first issue".

Reply:
Raised issue #1385